daprdocs/content/en/developing-applications/building-blocks/cryptography/howto-cryptography.md

```diff
@@ -15,14 +15,54 @@ Now that you've read about [Cryptography as a Dapr building block]({{< ref crypt
 
 ## Encrypt
 
-Using the Dapr gRPC APIs in your project, you can encrypt a stream of data, such as a file.
+{{< tabs "JavaScript" "Go" >}}
 
-{{< tabs "Go" >}}
+{{% codetab %}}
+
+<!--JavaScript-->
+
+Using the Dapr SDK in your project, with the gRPC APIs, you can encrypt data in a buffer or a string:
+
+```js
+// When passing data (a buffer or string), `encrypt` returns a Buffer with the encrypted message
+const ciphertext = await client.crypto.encrypt(plaintext, {
+    // Name of the Dapr component (required)
+    componentName: "mycryptocomponent",
+    // Name of the key stored in the component (required)
+    keyName: "mykey",
+    // Algorithm used for wrapping the key, which must be supported by the key named above.
+    // Options include: "RSA", "AES"
+    keyWrapAlgorithm: "RSA",
+});
+```
+
+The APIs can also be used with streams, to encrypt data more efficiently when it comes from a stream. In the example below, we are encrypting a file, writing to another file, using streams:
```

Comment:
```suggestion
The APIs can also be used with streams, to encrypt data more efficiently when it comes from a stream. The example below encrypts a file, writing to another file, using streams:
```

rewording to avoid using first person "we"